src/internal/work/work.go

```diff
@@ -388,3 +391,16 @@ func (w *Worker) subtaskFunc(subtaskKey string, processFunc ProcessFunc) subtask
 		}
 	}
 }
+
+// TaskCount returns how many subtasks are in the queue and how many are claimed.
+func (w *Worker) TaskCount(ctx context.Context) (int64, int64, error) {
```

Comment:
There should probably be names for the 2 int64s, so they can be distinguished by looking at the signature.  Use `_` for the error.